### PR TITLE
Add Bernd's slides + videos to website

### PR DESCRIPTION
**Pull request by HeidiSeibold**

### CODE REVIEW

File: vignettes/get_slide_data.R [hunk 1]
```diff
@@ -118,9 +126,11 @@ sldat$topic <- sapply(sldat$dir, generate_topic)
 names(sldat) <- capitalize(names(sldat))
 
 # Create links
-sldat$PDF <- paste0("[pdf](https://nbviewer.jupyter.org/github/compstat-lmu/lecture_i2ml/blob/master/slides-pdf/", sldat$Deck, ".pdf)")
-sldat$PDF[is.na(sldat$Youtube)] <- "Coming soon"
-sldat$YouTube <- paste0("[link](", sldat$Youtube, ")")
+# sldat$PDF <- paste0("[pdf](https://github.com/compstat-lmu/lecture_i2ml/blob/master/slides-pdf/", sldat$Deck, ".pdf)")
```
Comment (HeidiSeibold) [inline vignettes/get_slide_data.R]: Why was this changed the last time? I think relative links are simpler.

**Comment by HeidiSeibold** [comment]
@berndbischl can you take a look and merge if ok?

File: vignettes/get_slide_data.R [hunk 2]
```diff
@@ -128,6 +128,7 @@ names(sldat) <- capitalize(names(sldat))
 # Create links
 # sldat$PDF <- paste0("[pdf](https://github.com/compstat-lmu/lecture_i2ml/blob/master/slides-pdf/", sldat$Deck, ".pdf)")
 sldat$PDF <- paste0("[pdf](../../slides-pdf/", sldat$Deck, ".pdf)")
```
Comment (fabian-s) [inline vignettes/get_slide_data.R]: will these links work online as well? 
I used these nbviewer links so that the pdf slides are rendered nicely directly in the browser...

Comment (HeidiSeibold) [reply vignettes/get_slide_data.R]: I thought they would. But I can't test this now.

Comment (HeidiSeibold) [reply vignettes/get_slide_data.R]: In the end, we know your solution works, so let's switch again to that.

Comment (HeidiSeibold) [reply vignettes/get_slide_data.R]:
```suggestion
sldat$PDF <- paste0("[pdf](https://github.com/compstat-lmu/lecture_i2ml/blob/master/slides-pdf/", sldat$Deck, ".pdf)")
```

**Comment by berndbischl** [comment]
> @berndbischl can you take a look and merge if ok?

ehm, what am i exactly supposed to check? this PR touches 57 files, while many have nothing to do with what i did?

**Comment by berndbischl** [comment]
i mean it contains the chain of all of my recent commits, which is kinda clear, as I pushed them into the refactoring branch and you are now simply merging this into devel? so i dont really know whats expected here

**Comment by fabian-s** [comment]
.... just merge it @berndbischl :roll_eyes:
